src/main/java/org/chzzk/howmeet/domain/regular/room/entity/Room.java

```diff
@@ -33,8 +31,30 @@ public class Room extends BaseEntity {
     @Column(name = "description", nullable = false)
     private RoomDescription description;
 
+    @OneToMany(mappedBy = "room", cascade = CascadeType.ALL, orphanRemoval = true)
+    private List<MemberSchedule> schedules;
+
+    @OneToMany(mappedBy = "room", cascade = CascadeType.ALL, orphanRemoval = true)
+    private List<RoomMember> members;
+
     public Room(final RoomDescription description, final RoomName name) {
         this.description = description;
         this.name = name;
     }
+
+    public void updateDescription(RoomDescription description) {
+        this.description = description;
+    }
+
+    public void updateName(RoomName name) {
+        this.name = name;
+    }
+
+    public void setSchedules(List<MemberSchedule> schedules) {
+        this.schedules = schedules;
+    }
+
+    public void setMembers(List<RoomMember> members) {
+        this.members = members;
+    }
```

Comment:
setter는 아래와 같이 파라미터 검증이 있으면 좋을 것 같아요!

```suggestion
    public void updateName(final RoomName name) {
        if (!roomName.isNullOrEmpty()) {
            this.name = name;
        }
    }
```